docs/kss/starter-pages/details-1.html

```diff
@@ -0,0 +1,321 @@
+            <div class="y-navbar-center">
+                <div class="y-navbar-center-content">
+                    <h1>Page Title</h1>
+                </div>
+            </div>
+            <div class="navbar-right">
+            </div>
+        </div>
+    </div>
+
+    <!-- page level help -->
+    <span class="page-level-help-message">Page level help message goes here</span>
+
+    <!-- top panel -->
+    <div class="panel panel-default">
+        <div class="panel-body">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-4">
+                        <div class="y-thumbnail y-thumbnail--circle ng-scope" align="center">
+                            <img src="images/project-icon.png">
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <div class="row">
+                            <div class="col-md-12 form-readonly-text form-readonly-text-lg">
+                                <span class="badge badge-success">GOOD</span>
+                            </div>
+                            <div class="col-md-6">
+                                <div class="form-group">
+                                    <label>Label</label>
+                                    <p class="form-readonly-text form-readonly-text-lg">non editable value</p>
+                                </div>
+                                <div class="form-group">
+                                    <label>Label</label>
+                                    <input type="text" class="form-control input-lg">
+                                    <span class="help-inline help-inline--tooltip">
+                                    <span class="hyicon hyicon-helpoff"></span>
+                                    <span class="help-block-inline ng-binding">Additional help message goes here</span>
+                                    </span>
+                                    <span class="help-inline {$modifiers}">
+                                    <span class="help-block-inline">Start with a letter, use only alphanumeric characters 3-16 characters</span>
+                                    </span>
+                                    <p class="inline-hint-text"></p>
+                                </div>
+                            </div>
+                            <div class="col-md-6">
+                                <div class="form-group">
+                                    <label>Creation Date</label>
+                                    <p class="form-readonly-text form-readonly-text-lg">12/04/15</p>
+                                </div>
+                                <div class="form-group">
+                                    <label>Version</label>
+                                    <input type="text" class="form-control input-lg">
+                                    <span class="help-inline help-inline--tooltip">
```

Comment:
the indentation here for the children needs to be fixed. It's not clear that the elements inside of this page are children as they look like siblings